### PR TITLE
[MIG][10.0] purchase_last_price_info

### PR DESCRIPTION
**Pull request by Abranes**
As I didn't get an answer in #406 here, the migration to 10.0

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
@kittiu please check. Its not helpful to have 2 PRs for the same concept.

**Comment by Abranes** [comment]
Someone tested?

File: purchase_last_price_info/models/product.py [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
+                ('state', 'in', ['purchase', 'done'])
+            ], order='date_order DESC', limit=1)
+            product.last_purchase_date = line.date_order
+            product.last_purchase_price = line.price_unit
+            product.last_supplier_id = line.partner_id
 
     last_purchase_price = fields.Float(
-        string='Last Purchase Price', compute='_get_last_purchase')
+        string='Last Purchase Price',
+        compute='_get_last_purchase',
```
Comment (nikul-serpentcs) [inline purchase_last_price_info/models/product.py]: @Abranes IMO method name should be `_compute_last_purchase`.

same other

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
https://github.com/OCA/purchase-workflow/pull/587/commits/a2cf5f5b4f85d95b4e8a4ada147ac46165089e3e#diff-0deeef207fc91d4de1cf837a9fb5412eR13

`website` URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: purchase_last_price_info/models/product.py (hunk 1 above)
Comment (Abranes) [reply purchase_last_price_info/models/product.py]: @nikul-serpentcs Done!

**Comment by Abranes** [comment]
If anyone can help me, I don't know why travis fail, there is no error with my module and runbot is ok.

**Comment by pedrobaeza** [comment]
It is in your module:
```
purchase_last_price_info/__manifest__.py:13:80: E501 line too long (93 > 79 characters)
```

**Comment by Abranes** [comment]
Thank you @pedrobaeza, I didn't see it :sweat:

**Comment by pedrobaeza** [comment]
OK, no problem. I have to admit it's very hidden between the rest of the log lines.

File: purchase_last_price_info/models/product.py (hunk 1 above)
Comment (Abranes) [reply purchase_last_price_info/models/product.py]: @nikul-serpentcs Can you update your review?

**Review by CasVissers-360ERP — APPROVED** [review]
Functional test, LGTM.

**Comment by CasVissers-360ERP** [comment]
Only thing I would fix is the ability to handle multicurrency companies. The last price doesn't take the PO currency into account.

**Comment by CasVissers-360ERP** [comment]
In addition  ('product_id', '=', self.id), should be  ('product_id', '=', product.id),

**Pull request closed without merging**